system-addon/content-src/asrouter/components/ModalOverlay/_ModalOverlay.scss

```diff
@@ -11,7 +11,7 @@
   top: 0;
   width: 100%;
   display: none;
-  z-index: 100000;
+  z-index: 1000;
```

Comment:
Adjusting the z index here to not be so high - because it goes on top of the startup overlay for about:welcome